bagtrekkin/views.py

```diff
@@ -40,13 +41,19 @@ def search(request):
     if request.method == 'POST':
         search_form = SearchForm(request.POST)
         if search_form.is_valid():
-            passenger, luggages, logs = search_form.search()
-            context = {
-                'search_form': search_form,
-                'passenger': passenger,
-                'luggages': luggages,
-                'logs': logs
-            }
+            try:
+                passenger, luggages, logs = search_form.search()
+                context = {
+                    'search_form': search_form,
+                    'passenger': passenger,
+                    'luggages': luggages,
+                    'logs': logs
+                }
+            except ObjectDoesNotExist as e:
```

Comment:
Better use Model.DoesNotExist which is more closer and allows you to granularly control exceptions